### PR TITLE
Added information on query requests and new error for bad query params.

### PR DESCRIPTION
**Pull request by MorganEPatch**
API documentation update for timesync-node issue #129/PR #135.

### CODE REVIEW

File: source/draft_api.rst [hunk 1]
```diff
@@ -131,6 +131,21 @@ GET Endpoints
...
+parameters are used, they narrow down the result set (for example,
+`/times?user=example-user&activity=dev` will return all time entries which were entered by
+example-user AND which were spent doing development). When the same parameter is repeated,
+they expand the result set (for example, `/times?activity=gwm&activity=pgd` will return all
+time entries which were either for gwm OR pgd). Date ranges are inclusive on both ends.
+
+If a query parameter is provided with a bad value (e.g. invalid slug, or date not in ISO
+8601 format), a Bad Query Value error is returned. If a query parameter with a nonexistent
+name is provided, it is ignored. If multiple `start` or `end` parameters are provided,
+the first one sent is used. If a query parameter is not provided, it defaults to 'all values'
```
Comment (tschuy) [inline source/draft_api.rst]: end with a period.

File: source/draft_api.rst [hunk 2]
```diff
@@ -131,6 +131,21 @@ GET Endpoints
       "id": 1
     }
 
+In addition, the endpoint at `/times` also supports several querystring parameters: user,
```
Comment (tschuy) [inline source/draft_api.rst]: I think this needs double backticks.

File: source/draft_api.rst [hunk 3]
```diff
@@ -131,6 +131,21 @@ GET Endpoints
...
+project, activity, and date range. These are accessed via `/times?user=:username`,
+`/times?project=:projectslug`, `/times?activity=:activityslug`, `/times?start=:date`, and
+/times?end=:date (note that dates are in ISO-8601 format). When multiple different
```
Comment (Kennric) [inline source/draft_api.rst]: link to the ISO-8601  spec

File: source/draft_api.rst [hunk 4]
```diff
@@ -131,6 +131,21 @@ GET Endpoints
...
+parameters are used, they narrow down the result set (for example,
+`/times?user=example-user&activity=dev` will return all time entries which were entered by
+example-user AND which were spent doing development). When the same parameter is repeated,
+they expand the result set (for example, `/times?activity=gwm&activity=pgd` will return all
+time entries which were either for gwm OR pgd). Date ranges are inclusive on both ends.
+
+If a query parameter is provided with a bad value (e.g. invalid slug, or date not in ISO
+8601 format), a Bad Query Value error is returned. If a query parameter with a nonexistent
+name is provided, it is ignored. If multiple `start` or `end` parameters are provided,
```
Comment (Kennric) [inline source/draft_api.rst]: This is kind of ambiguous: "If a query parameter with a nonexistent name is provided, it is ignored."
Maybe: "Any query parameter other than those specified in this document will be ignored."

**Comment by Kennric** [comment]
Looks good. +1

**Pull request merged**